pkg/controller/kieapp/kieapp_controller.go

```diff
@@ -37,7 +40,11 @@ func Add(mgr manager.Manager) error {
 
 // newReconciler returns a new reconcile.Reconciler
 func newReconciler(mgr manager.Manager) reconcile.Reconciler {
-	return &ReconcileKieApp{client: mgr.GetClient(), scheme: mgr.GetScheme()}
+	imageClient, err := imagev1.NewForConfig(mgr.GetConfig())
+	if err != nil {
+		logrus.Error(err)
```

Comment:
Again, doesn't an error mean we can't move forward?

Reply:
i decided they DC should still deploy even if the imagestreamtag doesn't succeed. not sure what the benefit is of stopping the env deployment.

Reply:
Sorry, just seeing this now! My only concern was what we see much too often: a bunch of errors that are all red herrings, because the real errors was ignored earlier on. I guess that's not the case here, right? Because everything else will be successful, though the pods won't start, and the only error will be the one you log here?

Reply:
ah... maybe a warn instead? idk... ill think about other options.